apps/web/src/app/notifications/notification-page.tsx

```diff
@@ -19,7 +19,6 @@ export default function NotificationPage() {
   const client = useQueryClient();
   const mutation = useMutation({
     mutationFn: markNotificationsAsRead,
-    onSuccess() {},
```

Comment:
From https://tanstack.com/query/latest/docs/framework/react/reference/useMutation, `onSuccess` is optional.